src/main/java/com/overops/plugins/ConcoursePlugin.java

```diff
@@ -30,6 +29,7 @@ private ConcoursePlugin(String[] args) {
         try {
             Step<String[], QueryOverOps> prepStep = new PreparationStep(context);
             QueryOverOps query = prepStep.run(args);
+            context.getOutputStream().debugMode(query.isDebug());
```

Comment:
what exactly is this?

Reply:
If debug turned on Concourse plugin will show addition comments in the terminal.  
That corresponds property in Overops setup